react/src/base/inputs/Phone.stories.js

```diff
@@ -29,29 +29,90 @@ before submitting the form.
   to the hundredth place. Reformat user input
   with the \`formatter\` prop. This prop receives
   a function that takes the current value and returns
-  the reformatted value. 
+  the reformatted value.
 `,
   },
 };
 
 export const phoneInput = () => (
+  <SprkInputContainer>
+    <SprkLabel htmlFor="phone-1">Phone Number</SprkLabel>
+    <SprkInput id="phone-1" placeholder="(000) 000-0000" />
```

Comment:
We have type="tel" for phone in HTML. I checked Angular and its type=text. We've had convos in the past with design about type=tel vs. text for this particular thing. We should follow up with them.

Reply:
I asked, we'll add it in for React and Angular to match HTML. Should be there.